src/frontend/components/LiveVideoPanel/LiveVideoTabPanel/index.tsx

```diff
@@ -90,13 +90,11 @@ export const LiveVideoTabPanel = ({
   item,
   selected,
 }: LiveVideoTabPanelProps) => {
-  const intl = useIntl();
-
   return (
     <StyledTab
       title={
-        <StyledText data-testid="tab_title">
-          {intl.formatMessage(messages[item])}
+        <StyledText {...messages[item]}>
```

Comment:
FMI, why passing the message as props to StyledText ?

Reply:
because I did a mistake :(

Reply:
de8aaac